Ardagryd.js

```diff
@@ -288,7 +330,7 @@ class GridHeaderCell extends React.Component {
                 {label}
               </span>
               { sortable &&
-                <Button active={active} bsSize="xsmall" style={{marginLeft: '5px'}} onClick={this.sortChanged}>
+                <Button className={"sort-toggle-for-"+label} active={active} bsSize="xsmall" style={{marginLeft: '5px'}} onClick={this.sortChanged}>
```

Comment:
Isn't this going to cause trouble if `label` contains white-space? Why do we need a custom class here?